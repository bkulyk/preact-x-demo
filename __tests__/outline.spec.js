import React from 'react';
import Outline from '@bitchin/outline';
import { shallow } from './enzyme';
import { it, expect } from './jest'

it('should render correctly', () => {
  expect(shallow((
    <Outline
      children="children"
    />
  ))).toMatchSnapshot();
});
