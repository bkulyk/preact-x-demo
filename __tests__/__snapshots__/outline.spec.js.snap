// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="mdc-notched-outline"
>
  <div
    className="mdc-notched-outline__leading"
  />
  <div
    className="mdc-notched-outline__notch"
  >
    children
  </div>
  <div
    className="mdc-notched-outline__trailing"
  />
</div>
`;
